maquetado/styles.css

```diff
@@ -0,0 +1,27 @@
+body{
+  margin: 0;
+}
+
+.seccion-principal{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding-top:50px;
+  border-top:8px solid;
+  border-color: #f5c85f;
+
+}
+.separador{
+  width:3px;
+  height: 191px;
+  margin:0 30px;
+  background: linear-gradient(to bottom, transparent,#f5c85f,transparent);
+}
+
+/* Media queries */
+@media only screen and (max-width: 600px) {
+    .separador, .phone{
```

Comment:
when using a multiple selector with commas put every selector in a different line like this:
```
.separator, 
.phone {
  ...
}
```